flink-formats/flink-orc/src/test/java/org/apache/flink/orc/writer/OrcBulkWriterTest.java

```diff
@@ -48,14 +49,30 @@ public class OrcBulkWriterTest {
 
     @Test
     public void testOrcBulkWriter() throws Exception {
```

Comment:
Using @ParameterizedTest @EnumSource(CompressionKind.class) would be more elegant

Reply:
done